src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -334,8 +340,12 @@ private void launchApp(String itemMenuCode) {
 								((JDialog) target).setEnabled(true);
 							}
 						}
-					} catch (InstantiationException | ClassNotFoundException | IllegalAccessException ie) {
-						LOGGER.error("Error instantiating menu item: '{}' with class '{}'.", u.getCode(), u.getMyClass());
+					} catch (InstantiationException ie) {
+						ie.printStackTrace();
+					} catch (IllegalAccessException iae) {
+						iae.printStackTrace();
+					} catch (ClassNotFoundException cnfe) {
+						cnfe.printStackTrace();
```

Comment:
correct form is with multiple catch and LOGGER

Reply:
restored current informatici version